app/src/main/java/com/fmpdroid/moneybox/dto/Singleton.java

```diff
@@ -0,0 +1,32 @@
+package com.fmpdroid.moneybox.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Singleton {
+
+    private List<MoneyBoxDto> moneyBoxList;
+
+    private static Singleton instance;
+
+    private Singleton(){
+
+    }
+    public static Singleton getInstance() {
+        if (instance == null) {
+            instance = new Singleton();
+        }
+        return instance;
+    }
+
+    public void addMoneyBox(MoneyBoxDto moneybox) {
+        if (moneyBoxList == null){
+            moneyBoxList = new ArrayList<>();
+        }
+        moneyBoxList.add(moneybox);
+    }
+
+    public List<MoneyBoxDto> getMoneyBoxList(){
+        return moneyBoxList;
+    }
+}
```

Comment:
1. Move this class to a different directory. This is not a dto class. (funfact: know the difference between utility class and singleton class and when and how to use it. this is useful.)
2. remove the constructor and use the lombok's @NoArgsConstructor(access = AccessLevel.PRIVATE) annotation
3. rename the class into something meaningful - related to its purpose.